cmd/openshift/kodata/tekton-addon/0.0.1/01-clustertriggerbindings/bitbucket.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: triggers.tekton.dev/v1alpha1
```

Comment:
where `cmd/openshift/kodata/tekton-addon/0.0.1` `0.0.1` is a operator version 
Because all the CRD status have Version and for addons we don't have any version like `pipeline` and `trigger` so used operator version to indicate installed addons version

* If we want to avoid usage of operator version we can remove showing version for addons from CRD

Reply:
can we have different versions for different addons.
for eg:
```
clulstertasks/
    0.16/
            task1
            task2

webhook-extensions/
    0.1/
        reelase.yam
```

Reply:
i guess, for now we can merge this as it is and alter the ux later if necessary :+1: